src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -14591,5 +14575,199 @@ static int Test1(out int x)
 
             Assert.Equal("System.Int32", model.GetTypeInfo(yRef).Type.ToTestDisplayString());
         }
+
+        [Fact, WorkItem(13219, "https://github.com/dotnet/roslyn/issues/13219")]
+        public void IndexingDynamic()
```

Comment:
The thread is marked as resolved, but it doesn't look like there is a "specific test for type inference before the indexing is bound". What am I missing?

---
In reply to: [75763457](https://github.com/dotnet/roslyn/pull/13274#discussion_r75763457) [](ancestors = 75763457)